### PR TITLE
fix gesture tests on macos

### PR DESCRIPTION
**Pull request by danwalmsley**

### CODE REVIEW

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0026011-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

File: tests/Avalonia.IntegrationTests.Appium/GestureTests.cs [hunk 1]
```diff
@@ -147,9 +151,9 @@ public void RightTapped_Is_Raised_2()
             var device = new PointerInputDevice(PointerKind.Mouse);
             var b = new ActionBuilder();
 
-            b.AddAction(device.CreatePointerMove(border, 50, 50, TimeSpan.FromMilliseconds(100)));
+            b.AddAction(device.CreatePointerMove(border, 50, 50, TimeSpan.FromMilliseconds(50)));
             b.AddAction(device.CreatePointerDown(MouseButton.Right));
-            b.AddAction(device.CreatePointerMove(border, 2, 2, TimeSpan.FromMilliseconds(100)));
+            b.AddAction(device.CreatePointerMove(border, 52, 52, TimeSpan.FromMilliseconds(50)));
```
Comment (grokys) [inline tests/Avalonia.IntegrationTests.Appium/GestureTests.cs:156]: I think this should have been: `CreatePointerMove(CoordinateOrigin.Pointer...` here, which should move the pointer relative to its current position.

File: tests/Avalonia.IntegrationTests.Appium/GestureTests.cs [hunk 2]
```diff
@@ -118,12 +119,15 @@ public void DoubleTapped_Is_Raised_After_Control_Changes()
             // #8733
             var border = _session.FindElementByAccessibilityId("GestureBorder");
             var lastGesture = _session.FindElementByAccessibilityId("LastGesture");
-
+            
             new Actions(_session)
                 .MoveToElement(border)
                 .DoubleClick()
-                .DoubleClick()
```
Comment (grokys) [inline tests/Avalonia.IntegrationTests.Appium/GestureTests.cs:125]: Now this is just doing a one double-click, i think it's no longer testing #8733?

File: tests/Avalonia.IntegrationTests.Appium/GestureTests.cs (hunk 1 above)
Comment (danwalmsley) [reply tests/Avalonia.IntegrationTests.Appium/GestureTests.cs:156]: it says from the origin of the move... so I read that as origin of the border...

what I see on the screen is that with 52.. it moves 2 pixels


with 2 the test always fails.

File: tests/Avalonia.IntegrationTests.Appium/GestureTests.cs (hunk 2 above)
Comment (danwalmsley) [reply tests/Avalonia.IntegrationTests.Appium/GestureTests.cs:125]: its doing double click... having to move to get around appium being shit... then double click again,

**Comment by danwalmsley** [comment]
@grokys im going to merge, since this makes the integration tests now reliable... and we can check in a seperate PR the gesture tests.

**Pull request merged**